presto-docs/src/main/sphinx/sql/grant.rst

```diff
@@ -0,0 +1,44 @@
+=====
+GRANT
+=====
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    GRANT ( privilege [, ...] | ( ALL PRIVILEGES ) )
+    ON [ TABLE ] table_name TO ( grantee | PUBLIC )
+    [ WITH GRANT OPTION ]
+
+Description
+-----------
+
+Grants the specified privileges to the specified grantee.
+
+Specifying ``ALL PRIVILEGES`` grants ``DELETE, INSERT`` and ``SELECT`` privileges.
+
+Specifying ``PUBLIC`` grants privileges to all grantees.
+
+The optional ``WITH GRANT OPTION`` clause grants the privileges to grant the specified privileges.
```

Comment:
The optional ``WITH GRANT OPTION`` clause allows the grantee to grant these same privileges to other users.